agent/replacecall/src/main/resources/default-load-exclusions.txt

```diff
@@ -0,0 +1,21 @@
+// This file contains default package exclusions for the replacecall agent.
```

Comment:
Loading and transforming are closely intertwined. If you don't find the name helpful then it can be changed.